tests/e2e/e2e_common.go

```diff
@@ -175,7 +175,7 @@ const (
 	resizerContainerName                      = "csi-resizer"
 	scParamDatastoreURL                       = "DatastoreURL"
 	scParamFsType                             = "csi.storage.k8s.io/fstype"
-	scParamStoragePolicyID                    = "StoragePolicyId"
+	scParamStoragePolicyID                    = "storagePolicyID"
```

Comment:
should we do this for `DatastoreURL` and `StoragePolicyName` params as well?

Reply:
As I understood, this is the only changed param